projects/noloan/app/src/main/java/hasadna/noloan/mainactivity/SpamFragment.java

```diff
@@ -14,13 +14,14 @@
 import noloan.R;
 
 public class SpamFragment extends Fragment {
+  static final String TAG = "SpamFragment";
 
   private OnFragmentInteractionListener fragmentInteractionListener;
   private RecyclerView recyclerView;
-  private SpamRecyclerAdapter spamRecyclerAdapter;
+  SpamRecyclerAdapter spamRecyclerAdapter;
```

Comment:
Is it used somewhere?

Reply:
MainActivity - `updateTitles()`
Overall I think updating the titles can be done in a better way than it is at the moment, but for a later task...

Reply:
why main activity is the responsible to update the title?
maybe add a update title in each fragment and main activity just call it, so mainActivity don't need to know about the recycler.